python/lsst/ip/isr/isrTask.py

```diff
@@ -1777,7 +1783,15 @@ def run(self, ccdExposure, *, camera=None, bias=None, linearizer=None,
         outputBin1Exposure = None
         outputBin2Exposure = None
         if self.config.doBinnedExposures:
-            outputBin1Exposure, outputBin2Exposure = self.makeBinnedImages(ccdExposure)
```

Comment:
Can you also add a `@deprecated` decorator to the `makeBinnedImages` method for both tasks as well?  I don't think anyone else is using it, but this will at least warn people before it goes away.